### PR TITLE
Remove warnings related to CA1707

### PR DESCRIPTION
**Pull request by lahma**
* Remove the underscores from member name

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Email.Azure/Services/AzureEmailProviderBase.cs [hunk 1]
```diff
@@ -16,7 +16,9 @@ public abstract class AzureEmailProviderBase : IEmailProvider
     // Common supported file extensions and their corresponding MIME types for email attachments
     // using Azure Communication Services Email.
     // For more info <see href="https://learn.microsoft.com/en-us/azure/communication-services/concepts/email/email-attachment-allowed-mime-types" />
+#pragma warning disable CA1707 // Remove the underscores from member name
     protected static readonly Dictionary<string, string> _allowedMimeTypes = new()
```
Comment (Piedone) [inline src/OrchardCore.Modules/OrchardCore.Email.Azure/Services/AzureEmailProviderBase.cs:20]: This should actually be `AllowedMimeTypes` anyway.

**Review by Piedone — CHANGES_REQUESTED** [review]

File: src/OrchardCore.Modules/OrchardCore.Email.Azure/Services/AzureEmailProviderBase.cs (hunk 1 above)
Comment (lahma) [reply src/OrchardCore.Modules/OrchardCore.Email.Azure/Services/AzureEmailProviderBase.cs:20]: It's OK as a breaking change to change this?

Comment (Piedone) [reply src/OrchardCore.Modules/OrchardCore.Email.Azure/Services/AzureEmailProviderBase.cs]: Yes, because this code wasn't released yet.

Comment (lahma) [reply src/OrchardCore.Modules/OrchardCore.Email.Azure/Services/AzureEmailProviderBase.cs]: pushed a commit to change the name to be upper-case

**Review by Piedone — APPROVED** [review]

**Review by hishamco — APPROVED** [review]

**Pull request merged**